README.md

```diff
@@ -1,5 +1,15 @@
 # 02-OKJK
-Private repo for work by team OKJK
+
+Summary of Problem: There are a lot of economic indicators being captured by the Bureau of Labor Statistics (BLS). However, there is only a small set of the population that understands what these indicators mean. OKJK found common units between the data they were given by BLS. What they found was a relationship between the age, unemployment rates, and the price of an average grapefruit with the economic score of that state. As age, unemployment, and the price of a grapefruit increase, states’ economic scores decrease (this is reversed in the application due to an error in the model code). To demonstrate this, OKJK created an interactive web application that rates economic development based on user input relating to age, unemployment rate, and price of an average grapefruit. OKJK uses requests to get data from the Bureau. OKJK implements a pandas DataFrame which allows them to quickly combine, normalize, and fill in data. Their website runs on Heruko, a platform that allows developers to run applications entirely in the cloud. It uses Vincent for visualization and Flask for its framework. 
```

Comment:
Cool! @Capez can you include the above info and also host the diagram images in the repo as well for posterity in another PR :+1: